lib/client/user/mediator-topics/clear_session.js

```diff
@@ -0,0 +1,32 @@
+var CONSTANTS = require('../../../constants');
+
+/**
+ * Initialsing a subscriber for clearing a user session
+ *
+ * @param {object} userEntityTopics
+ * @param {UserClient} userClient - The User Client
+ */
+module.exports = function clearSession(userEntityTopics, userClient) {
+
+  /**
+   *
+   * Handling clearing a user session
+   *
+   * @param {object} parameters
+   * @param {string/number} parameters.topicUid  - (Optional)  A unique ID to be used to publish completion / error topics.
+   * @returns {*}
+   */
+  return function handleClearSession(parameters) {
+    var self = this;
+    parameters = parameters || {};
+    var clearSessionDoneTopic = userEntityTopics.getTopic(CONSTANTS.TOPICS.CLEAR_SESSION, CONSTANTS.DONE_PREFIX, parameters.topicUid);
```

Comment:
We should not longer use this form. Mediator topics have done,error support outside the box.